### ISSUE TITLE
[PL][Guild][Achievement] United Nations

### ISSUE THREAD

**Issue by Invaerne** [Atlantiss/BugTracker#1887]
Prosze nie zamykac tego tematu, wiem, ze juz taki byl i zostal zamkniety, ale chcialabym sprobowac wkleic go jeszcze raz. Achievement nie dziala tak, jak powinien, a przynajmniej w gildii Legion.
http://www.wowhead.com/achievement=5892/united-nations
![achie1](https://cloud.githubusercontent.com/assets/10252300/5966193/7c297d24-a802-11e4-8c0c-8f451cdda10e.jpg)
![achie2](https://cloud.githubusercontent.com/assets/10252300/5966194/815f304a-a802-11e4-9f1a-4fba9855b74a.jpg)
Jak widac mam wbite exa w gildii, wiec przynajmniej wszystkie moje repy powinny byc zaliczone, kiedy jestem online (bo jest teoria, ze zalicza tylko repy ludzi, ktorzy sa online). Ale nie zalicza jednak kilku moich exaltedow, a konkretnie Sporregar, Kalu'ak, Ramkahen i Earthen Ring. Oto dowody:
![achie3](https://cloud.githubusercontent.com/assets/10252300/5966223/cc3da98e-a802-11e4-94bd-3d91e44a3a5a.jpg)
![achie4](https://cloud.githubusercontent.com/assets/10252300/5966224/d0d9cb80-a802-11e4-9d0f-85ca92cc7e6f.jpg)
![achie5](https://cloud.githubusercontent.com/assets/10252300/5966229/d56777ec-a802-11e4-9978-699540a70d78.jpg)
Relogi ani odczekanie nic nie daje. Dodam, ze Guild Master ma wbity tytul The Exalted (cos mowil, ze ma 45 czy 55 rep) i na 100% ma exalted w gildii, a mimo wszystko nie zalicza nawet tych wymaganych 40 exa. Nie wiem, jak jest w innych gildiach, moze to byc randomowy bug, ale szkoda, ze nie mozna przez niego zdobyc gildyjnej nagrody.

**Comment by Drutmaster** [comment]
Jest podobnie, u nas również brak wielu reputacji, mimo że na 4.0.6 mieliśmy achievement zaliczony. Sam posiadam 54 exalted i większości z nich achiv nie naliczył.

**Comment by Invaerne** [comment]
Achievement nadal nie dziala, niektorych rep po prostu nie zalicza.

**Comment by Bastion** [comment]
Ktoś coś mógłby zaglądnąć w te gildyjne acziwy bo za bossy też nie działają

**Comment by Illusionist87** [comment]
Were you people exalted with your guild before getting exalted with those factions?
If not that might be the problem.
On a similar guild achievement "Working as a team" (where you need honored rep with guild and 525 profession), if you got 525 on  profession first and than the rep it wouldn't count for the achievement. You needed to get the rep first and than level the profession.

It might be the same problem here. If someone could confirm or correct my assumption, it would be great. :D

**Comment by Alauree** [comment]
Witam, prawdopodobnie @op odniósł się do tematu, który ja bardzo dawno temu założyłem. Cała sprawa polega na tym, że od przejścia z 4.0.6 na 4.3.4 achiv całkowicie został zresetowany, przez co każda gildia musiała wbijać każdą repę od nowa. Jest to bardzo problematyczne, gdyż do tego achiva trzeba wbijac repki takie jak ravenholdt, które wymagają BARDZO dużo czasu...

**Comment by Hartwall** [comment]
@Illusionist87 This is correct, getting the reputation on a faction exalted before getting guild exalted makes this happen. Looking at wowhead comments relogging seems to trigger a check for this but it doesnt.
